radio/src/translations/en.h

```diff
@@ -65,6 +65,8 @@
 #define TR_VVARIOCENTER                "Tone","Silent"
 #define TR_CURVE_TYPES                 "Standard","Custom"
 
+#define TR_CRSF_ARMING_MODES           "Channel","Function"
```

Comment:
Should 'Channel' option be 'Channel5' to make the intent clearer?
Also this is missing in other language files.